src/modules/0.0.16/aws_ecs_simplified/index.ts

```diff
@@ -48,7 +48,7 @@ const prefix = 'iasql-ecs-';
 
 // TODO: remove this once the aws gateway gets disolved
 async function getAwsClient(ctx: Context): Promise<AWS> {
-  const awsCreds = await ctx?.orm?.findOne(AwsAccount.mappers.awsAccount.entity);
+  const awsCreds = await ctx?.orm?.findOne(awsAccount.awsAccount.entity);
```

Comment:
Maybe it should be renamed to `awsAccountModule` to have something like `awsAccountModule.awsAccount.entity`

Reply:
For most it's not a huge deal because it's things like `AwsEc2.instance...` and only modules that have a single mapper is this an issue. I can make that change through renaming during import?